sirepo/package_data/static/js/flash.js

```diff
@@ -9,8 +9,24 @@ SIREPO.app.config(function() {
     SIREPO.SINGLE_FRAME_ANIMATION = ['gridEvolutionAnimation'];
 });
 
-SIREPO.app.factory('flashService', function(appState) {
+SIREPO.app.factory('flashService', function(appState, $rootScope) {
     var self = {};
+    self.originalGridgeometryEnum = [];
+    self.capLaserBELLAGridgeometyEnum = [];
+
+    function initGridgeometry() {
+        if (self.originalGridgeometryEnum.length) {
+            return;
+        }
+        var e = SIREPO.APP_SCHEMA.enum.Gridgeometry;
+        self.originalGridgeometryEnum = e;
+        e.forEach(function(v) {
+            if (self.isFlashType('CapLaserBELLA') && ['polar', 'spherical'].includes(v[0])) {
```

Comment:
We've been using Array.indexOf() rather than Array.includes() because MSIE doesn't support that method.